src/modules/oaf/components/OafMask.js

```diff
@@ -334,7 +334,7 @@ export class OafMask extends MvuElement {
 						<span id="oaf-title" class="text">${layerProperties.title ? layerProperties.title : translate('oaf_mask_title')}</span>
 						${getInfoBarHtml()}
 					</h3>
-					${tabs()}
+					<div class="header__buttons">${tabs()}</div>
```

Comment:
Could we extend or add a test for this class name?

Reply:
done